### DISCUSSION TITLE
Can a user have many accesstokens?

### DISCUSSION THREAD

**Discussion by taikon** [fastapi-users/fastapi-users#1147] [Q&A]
Can a user have many accesstokens? I am using the database strategy.

**Comment by frankie567** [comment]
Yes, nothing prevents that. For example, if a user authenticates on a first computer, then on another one, this user will have two access tokens in database.

**Comment by taikon** [comment]
Thanks. And how would I be able to retrieve a list of all of a user's tokens? I tried the following but it didn't work. 
```
@app.get("/authenticated-route")
async def authenticated_route(user: User = Depends(current_active_user)):
    return {"message": f"tokens: {user.tokens}"}
```

**Reply by frankie567** [reply]
Yes, we don't load associated tokens of a user by default.

What you need to do is to build your own SQLAlchemy query for `AccessToken` to retrieve them.
